src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/IPersonGroupOperations.cs

```diff
@@ -91,21 +91,21 @@ public partial interface IPersonGroup
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<HttpOperationResponse<PersonGroupResult>> GetWithHttpMessagesAsync(string personGroupId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<HttpOperationResponse<PersonGroup>> GetWithHttpMessagesAsync(string personGroupId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
         /// <summary>
         /// Update an existing person group's display name and userData. The
         /// properties which does not appear in request body will not be
         /// updated.
         /// </summary>
         /// <param name='personGroupId'>
-        /// personGroupId of the person group to be updated.
+        /// Id referencing a particular person group.
         /// </param>
         /// <param name='name'>
-        /// Name of the face list, maximum length is 128.
+        /// Name of the face collection, maximum length is 128.
         /// </param>
         /// <param name='userData'>
-        /// Optional user defined data for the face list. Length should not
-        /// exceed 16KB.
+        /// Optional user defined data for the face collection. Length should
```

Comment:
Ditto. Same error. Not face collection here. Please also check other places. No similar comments again.

Reply:
Done